addons/stock/models/product.py

```diff
@@ -583,6 +583,17 @@ def _get_rules_from_location(self, location, route_ids=False, seen_rules=False):
         else:
             return self._get_rules_from_location(rule.location_src_id, seen_rules=seen_rules | rule)
 
+    def _get_date_with_security_lead_days(self, date, location):
```

Comment:
would set this method on res.company instead

Reply:
The comment is for _get_security_by_rule_action